client/src/Components/OneVideoCard.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+const OneVideoCard = ({ id, title, url, rating, handleDelete }) => {
+  const [videoRating, setVideoRating] = useState(rating);
+
+  const increase = () => {
+    setVideoRating(videoRating + 1);
```

Comment:
FYI, setVideoRating(rating => rating + 1) also works, this approach it wont need the videoRating var on line number 3 as the set function can get that from the param. both approaches work in this case

Reply:
Thanks for the review :)